app/services/idv/phone_confirmation_otp_generator.rb

```diff
@@ -0,0 +1,8 @@
+module Idv
+  class PhoneConfirmationOtpGenerator
+    def self.generate_otp
```

Comment:
What do you think about calling this method `call` for consistency with other service objects, and for less redundancy with the class name?

Reply:
I don't mind

Reply:
Or maybe use a verb for the class name: `GeneratePhoneConfirmationOtp`?